tests/test_playbooks/organization.yml

```diff
@@ -20,6 +20,24 @@
       expected_diff: true
       expected_diff_before: ""
       expected_diff_after: "description.*test"
+  - include: tasks/organization.yml
+    vars:
+      organization_state: present
+      organization_name: "会社"
+      organization_description: "アクメ株式会社"
+      expected_change: true
+      expected_diff: true
+      expected_diff_before: ""
+      expected_diff_after: "description['\": ]*\\\\u30a2\\\\u30af\\\\u30e1\\\\u682a\\\\u5f0f\\\\u4f1a\\\\u793e"
```

Comment:
I don't think using `expected_diff` here brings a lot of benefit to the tests (besides ugly unicode nunmbers), but I'd also not block on this.

Reply:
I agree that the `expected_diff_after` regexp is rather ugly, but I think the main features of this test are that the module does not crash when unicode characters are used and that the result the module creates is also what was filled in.

What came to my mind now is that the description is already being checked in the next test.
So I could change this test to check the resulting `organization_name` instead, which would also reduce the number of unicodes that need to be checked.

Reply:
that sounds good!